armory/scenarios/main.py

```diff
@@ -208,7 +208,7 @@ def run_config(*args, **kwargs):
     args = parser.parse_args()
     update_filters(args.log_level, args.debug)
     log.trace(f"main.py called update_filters({args.log_level} debug: {args.debug})")
-    calling_version = os.getenv(environment.ARMORY_VERSION, "UNKNOWN")
+    calling_version = os.environ.get("ARMORY_VERSION", "UNKNOWN")
```

Comment:
I'm not a fan of replacing `environment.ARMORY_VERSION` with a static string `"ARMORY_VERSION"` everywhere. That makes it more difficult to change in the future.

Instead, let's just rename the attribute in environment to be more understandable, like:
```
CLIENT_ARMORY_VERSION = "ARMORY_VERSION"
```
and then use `CLIENT_ARMORY_VERSION` throughout.